app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
```

Comment:
- [ ] Please remove the class_name and foreign_key attributes from this association because it's already known for the `Comment` model that it belongs_to the `Post` model, and the post's field name in the database is `post`, so it's PK will be automatic `post_id`.